packages/react-native-editor/__device-tests__/gutenberg-editor-file-@canary.test.js

```diff
@@ -2,35 +2,35 @@
  * Internal dependencies
  */
 import { blockNames } from './pages/editor-page';
+import { waitForMediaLibrary } from './helpers/utils';
 import testData from './helpers/test-data';
-import { isAndroid } from './helpers/utils';
 
 describe( 'Gutenberg Editor File Block tests', () => {
 	it( 'should be able to add a file block', async () => {
```

Comment:
With the consolidation of both test cases, I'm wondering if we should include here the description of the removed one (i.e. adding a file to the block), wdyt?

Reply:
ouch, good point. Addressed with eef087b6fa3da691f0f9258381091226b21bd2e1.